src/Transports/MassTransit.AmazonSqsTransport/AmazonSqsTransport/Configuration/AmazonSqsRegistrationBusFactory.cs

```diff
@@ -50,7 +51,7 @@ public override IBusInstance CreateBus(IBusRegistrationContext context, IEnumera
                         h.SecretKey(options.SecretKey);
                     }
                     else
-                        h.Credentials(FallbackCredentialsFactory.GetCredentials());
+                        h.Credentials(DefaultAWSCredentialsIdentityResolver.GetCredentials());
```

Comment:
In V4 `FallbackCredentialsFactory` was marked obsolete because it's credential search path was inconsistent with other AWS SDKs. The new `DefaultAWSCredentialsIdentityResolver` type now uses the same credential search path as other SDKs.